### PR TITLE
Fix/5954 5952

### LINKED ISSUE #5952: Provisioner: network is hidden by default
**Describe the bug**
When you configure a provisioner to connect to a SSID, default value for `broadcast` is `0` which means SSID is hidden and description is:

Capture d’écran de 2020-11-02 16-53-11

From my point of view, it's really contre-intuitive.

**To Reproduce**
1. Create a new provisioner

**Expected behavior**
By default, `broadcast` should be set to `1`: SSID is not hidden. If you want to configure a hidden SSID, you have to uncheck button.

### LINKED ISSUE #5954: web admin(portal modules): with_aup default value is 1
**Describe the bug**
When you create a new `Authentication::Login` portal module, GUI display `Require AUP` button as disabled even if API send a default value of `1`.

**To Reproduce**
1. Create a new `Authentication::Login` portal module without changing defaults

=> Require AUP is disabled

2. Check value sent by frontend to backend

=> `with_aup=1`

This value is saved in `portal_modules.conf`

**Expected behavior**
Display `Require AUP` as enabled on frontend.

### PR DESCRIPTION
**Pull request by nqb**
# Description
- SSIDs are not hidden by default when creating a provisioner (#5952) 
- with_aup is correctly displayed on GUI (#5954)

# Impacts
Provisioning
Portal Module

# Issue
fixes #5952 
fixes #5954 

# Delete branch after merge
YES

# NEWS file entries
## Bug Fixes
* SSIDs are not hidden by default when creating a provisioner (#5952) 
* with_aup is correctly displayed on GUI (#5954)

### CODE REVIEW

**Comment by nqb** [comment]
`with_aup` fix works but not `broadcast` fix.

**Comment by nqb** [comment]
Tested after build on a pipeline and both settings works as expected.

**Comment by nqb** [comment]
Bump

**Review by satkunas — APPROVED** [review]
Javascript looks good!

**Comment by nqb** [comment]
@jrouzierinverse, is it ok for you ?

File: html/pfappserver/lib/pfappserver/Form/Config/PortalModule/Authentication.pm [hunk 1]
```diff
@@ -35,7 +35,8 @@ has_field 'with_aup' =>
   (
    type => 'Checkbox',
    label => 'Require AUP',
-   checkbox_value => '1',
+   checkbox_value => 1,
+   unchecked_value => 0,
```
Comment (jrouzierinverse) [inline html/pfappserver/lib/pfappserver/Form/Config/PortalModule/Authentication.pm]: Rename unchecked_value -> input_without_param 
https://metacpan.org/pod/HTML::FormHandler::Field::Checkbox

File: html/pfappserver/lib/pfappserver/Form/Config/Provisioning/mobileconfig.pm [hunk 2]
```diff
@@ -30,9 +30,10 @@ has_field 'broadcast' =>
    type => 'Checkbox',
    label => 'Broadcast network',
    checkbox_value => 1,
-   input_without_param => 0,
+   unchecked_value => 0,
```
Comment (jrouzierinverse) [inline html/pfappserver/lib/pfappserver/Form/Config/Provisioning/mobileconfig.pm]: Rename unchecked_value -> input_without_param

https://metacpan.org/pod/HTML::FormHandler::Field::Checkbox

**Review by jrouzierinverse — CHANGES_REQUESTED** [review]
unchecked_value is not valid parameter.

**Comment by nqb** [comment]
Thanks @jrouzierinverse, it should be good now.

**Pull request merged**